### PR TITLE
Update styles for send screen

### PR DESCRIPTION
**Pull request by x-goose-x**
### Summary:
A number of style changes were required to reflect the new wallet design flow. This pull request addresses many of those style changes, largely to ensure closer consistency with design and to add a few more features, like translucencies on the viewfinder.

screenshot_20170926-144617

Still working on adding torch/flashlight functionality, and this screen will need choose contacts.

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/wallet/send/styles.cljs [hunk 1]
```diff
@@ -49,14 +50,18 @@
...
-   :margin-horizontal 28
-   :margin-vertical   10
-   :border-radius     5
-   :background-color  styles/color-blue5})
+  (cond-> {:flex-direction    :column
+           :margin-horizontal 28
+           :margin-vertical   20
+           :border-radius     8
+           :background-color styles/color-blue6}
+    platform/android? (dissoc :background-color)))
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/send/styles.cljs]: please use `defstyles` macro

File: src/status_im/ui/screens/wallet/send/styles.cljs [hunk 2]
```diff
@@ -49,14 +50,18 @@
...
 
 (def recipient-icon {:margin-right 20})
 
+(def recipient-icon-disabled {:margin-right 20
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/send/styles.cljs]: indentation

File: src/status_im/ui/screens/wallet/send/styles.cljs [hunk 3]
```diff
@@ -65,37 +70,83 @@
 
 (def recipient-button-text
   {:color     :white
-   :font-size 17})
+   :align-self :center
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/send/styles.cljs]: indentation

File: src/status_im/ui/screens/wallet/send/styles.cljs [hunk 4]
```diff
@@ -65,37 +70,83 @@
...
+   :font-size 14})
+
+(def recipient-button-text-disabled
+  (merge recipient-button-text {:color "rgba(255, 255, 255, 0.3)"}))
 
 (defn recipient-touchable [divider?]
-  (cond-> {:border-color styles/color-gray-transparent-medium-light}
-    divider? (assoc :border-bottom-width 1)))
+  (cond-> {:border-color styles/color-gray-transparent-light}
```
Comment (flexsurfer) [inline src/status_im/ui/screens/wallet/send/styles.cljs]: `defnstyle` macro

File: src/status_im/translations/en.cljs [hunk 5]
```diff
@@ -375,7 +375,7 @@
    :transactions-filter-type              "Type"
    :transactions-filter-select-all        "Select all"
    ;; Wallet Send
-   :wallet-send-transaction               "Send Transaction"
+   :wallet-send-transaction               "Send a Transaction"
```
Comment (flexsurfer) [inline src/status_im/translations/en.cljs:378]: is it correct with article?

**Review by flexsurfer — CHANGES_REQUESTED** [review]

File: src/status_im/translations/en.cljs (hunk 5 above)
Comment (x-goose-x) [reply src/status_im/translations/en.cljs:378]: There's disagreement between ios and android screens at the moment.

Comment (flexsurfer) [reply src/status_im/translations/en.cljs:378]: designers should fix mockups

**Review by flexsurfer — APPROVED** [review]

**Pull request merged**